recipes-tag/03-id-hxn-collection/meta.yaml

```diff
@@ -7,10 +7,11 @@ package:
   version: {{ year }}{{ cycle }}.{{ version }}
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   run:
+    - hxnfly ==0.0.2
```

Comment:
I suggest changing this to `>=0.0.2` so that we don't need remember to update this when `hxnfly` updates.